tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/tabbed_paragraph_block.html

```diff
@@ -0,0 +1,36 @@
+{% load wagtailcore_tags %}
```

Comment:
It might be simplest to update this to very simple output that just shows the variable names, in order to avoid merge conflict with the front-end template.